src/hacbs/components/ApplicationDetails/ApplicationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { TextVariants, Text, Button, Modal, TextContent } from '@patternfly/react-core';
+import imageUrl from '../../imgs/application-overview.svg';
+
+export const HACBS_APPLICATION_MODAL_KEY = 'hacbs/showApplicationModal';
+
+interface ApplicationModalProps {
+  showApplicationModal: boolean;
+  onClose: () => void;
+}
+
+const ApplicationModal: React.FC<ApplicationModalProps> = ({ showApplicationModal, onClose }) => {
+  const footer = (
+    <div className="pf-u-display-flex pf-u-justify-content-center pf-u-flex-fill">
+      <Button
+        data-testid="hacbs-getting-started-modal-dismiss"
+        aria-label="get started"
```

Comment:
Wrong label?
I don't think an aria label is necessary here as the button itself has valid text.